pfb-analysis/scripts/setup_database.sh

```diff
@@ -6,11 +6,18 @@ NB_POSTGRESQL_DB=pfb
 NB_POSTGRESQL_USER=gis
 NB_POSTGRESQL_PASSWORD=gis
 
+# Set configuration parameters
+su postgres bash -c psql <<EOF
+ALTER SYSTEM SET work_mem TO '4096MB';
```

Comment:
Do these actually take effect? The docs states: "Values set with ALTER SYSTEM will be effective after the next server configuration reload, or after the next server restart in the case of parameters that can only be changed at server start. A server configuration reload can be commanded by calling the SQL function pg_reload_conf(), running pg_ctl reload, or sending a SIGHUP signal to the main server process."

I don't think we reload the pg conf, and I don't see that it happens anywhere in the base container's [documentation notes](https://hub.docker.com/_/postgres/).